lib/rules/indent.js

```diff
@@ -375,16 +375,39 @@ module.exports = {
         }
 
         /**
+         * Returns a boolean if the parent type is in the nodeList
+         * @param {string} parentType type that is being checked in nodeList
+         * @param {array} nodeList list of node types passed in
+         * @returns {boolean} if parentType matches item in nodeList, otherwise false
+         */
+        function checkIfParentTypeIsInStopNodeList(parentType, nodeList) {
```

Comment:
This is a minor nit and shouldn't block this PR from being merged, but I think this function can be replaced with just `nodeList.indexOf(parentType) !== -1`.